### PR TITLE
added method to get all player countries that are connected to the co…

### PR DESCRIPTION
**Pull request by Shashaank5**
…untry passed in

### CODE REVIEW

File: src_test/RiskModelTest.java [hunk 1]
```diff
@@ -153,6 +153,12 @@ public void testDefenderLostCountry(){
         assertEquals(player1 ,India.getPlayer());
     }
```
Comment (Shashaank5) [inline src_test/RiskModelTest.java:155]: This test case is not finished. Just for testing that the method doesn't crash

File: src/RiskModel.java [hunk 2]
```diff
@@ -127,6 +127,23 @@ public void attack(Country attackingCountry, Country defendingCountry, int numOf
...
+        countryRecurse(adjacentCountries, country);
+        return adjacentCountries;
+
+    }
+
+    private void countryRecurse(ArrayList playerCountries, Country country){
+        playerCountries.add(country);
+        for (Country adjacent: country.getAdjacentCountries()){
+            //if owned by same player and not in list already add it
+            if(adjacent.getPlayer() == ((Country) playerCountries.get(0)).getPlayer() && !playerCountries.contains(adjacent)) {
```
Comment (Justgo13) [inline src/RiskModel.java:141]: This should probably just be the country parameter instead of hardcoding .get(0)

**Review by Justgo13 — CHANGES_REQUESTED** [review]

**Review by Justgo13 — CHANGES_REQUESTED** [review]
Remove hard coded .get(0)

**Comment by Shashaank5** [comment]
> Remove hard coded .get(0)

This accesses the first country so it is necessary unless the parameters of the function are changed

**Pull request merged**